tests/test_token_regression.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import os
+from os.path import exists
+
+from subscriber import podaac_access as pa
+import shutil
+from pathlib import Path
+
+@pytest.mark.regression
+def setup_module(module):
+    print('*****SETUP*****')
+    tokens = pa.list_tokens(pa.token_url)
+    for x in tokens:
+        pa.delete_token(pa.token_url, x)
+
+@pytest.mark.regression
+def teardown_module(module):
+    print('*****TEARDOWN*****')
+    tokens = pa.list_tokens(pa.token_url)
+    for x in tokens:
+        pa.delete_token(pa.token_url, x)
+
+# REGRESSION TEST CURRENTLY REQUIRES A .NETRC file for CMR/Data Download
+# token API can be found here: https://wiki.earthdata.nasa.gov/display/EL/API+Documentation
+# explore https://urs.earthdata.nasa.gov/documentation/for_integrators/api_documentation#/oauth/token
+@pytest.mark.regression
+def test_list_tokens():
+    tokens = pa.list_tokens(pa.token_url)
+    for x in tokens:
+        pa.delete_token(pa.token_url, x)
```

Comment:
Not sure what this is testing? Just that no exceptions are thrown?

Reply:
Updated to better test this.